configs/test/project.yaml

```diff
@@ -96,3 +96,11 @@ env:
   # input to fuzzer as part of --input-dir argument (disabled by default).
   # WEB_TESTS_URL: 'gs://bucket/path-to-tests-archive.zip'
   # UPDATE_WEB_TESTS: true
+
+project_setup:
```

Comment:
Do we want to comment this block since it wont apply to any other user unless they need project setup and they need to create these buckets manually (outside of create_config). So, can add a comment that this is for XYZ usecase where you want automatic project setup and need to create custom buckets ?

Reply:
Also, do we need to put a commented cron entry for this, since this project-setup cron is not listed in test config.

Reply:
Hmm, I'm not sure if this is a use case many others will have, so I don't think documenting this/commented cron is necessary.

I added these here for easier testing, but I can comment this out in another PR.

Reply:
Nice, in that case, just remove this in a future cl.